libraryValidations/Python/test_json_validations_with_provider.py

```diff
@@ -3,6 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for
```

Comment:
I still feel like this code should be deduped between provider and no provider. Ideally the only difference is where we're loading flags from and which tests we read.

Reply:
I think the issue is that the telemetry part at the very end, I could try to rework it, but that could be it's own PR.